plugins/kotlin/idea/tests/testData/editor/enterHandler/controlFlowConstructions/ElseInWhenWithoutOption.after.kt

```diff
@@ -2,8 +2,8 @@ fun some() {
     when {
         true && false ->
         else
-        <caret>-> Unit
+            <caret>-> Unit
```

Comment:
This test-case is the last one thing I can't figure out.
It always fails:
- when I add indentation it says:
   ```
   expected:<...       else
       []<caret>-> Unit
   }...> but was:<...       else
       [    ]<caret>-> Unit
   }...>
   ```
- when I remove indentation it says:
   ```
   expected:<...       else
       [    ]<caret>-> Unit
   }...> but was:<...       else
       []<caret>-> Unit
   }...>
   ```

Reply:
@osipxd Could you share the exact error message?

Reply:
> What does it represent? space? tab or normal_space?
It should be `NORMAL_INDENT`, I think. The problem is that actual result changes when I change expected result (without any changes in code). I'm not quite familiar with these tests so maybe it is expected behaviour. Maybe it runs assertion twice before and after reformatting?

Reply:
@dimonchik0036, can you help, please?

Reply:
The problem persists. This test always fails.

Reply:
This one is fixed in commit 975c2656445ddd234c566a7c70becc35452eade6